### PR TITLE
SPEC.md: make the fields consistent

### PR DESCRIPTION
**Pull request by vbatts**
/cc @jonboulle

### CODE REVIEW

File: SPEC.md [hunk 1]
```diff
@@ -543,6 +543,8 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
 
 ### AC Name Type
 
+Field `acName`
```
Comment (jonboulle) [inline SPEC.md]: This isn't really correct, there's no such field acName...

Comment (vbatts) [inline SPEC.md]: ohman. It's app/name. `acName` completed on my editor. hmm

Comment (jonboulle) [inline SPEC.md]: Hrmmm but still, the AC Name type is used much more broadly than for the app name..

Comment (vbatts) [inline SPEC.md]: what are you saying? just remove acName here? since it's used else where as a formatting strictness for field values

Comment (jonboulle) [inline SPEC.md]: I guess I am unclear what you are trying to point out with these "field" references - in the case of acKind and acVersion it is a simple 1:1 between the type and the field name, but in the case of AC Name it's a more generic type used by a whole bunch of fields...

Comment (jonboulle) [inline SPEC.md]: So maybe just removing it here makes sense.

Comment (vbatts) [inline SPEC.md]: k

**Comment by vbatts** [comment]
updated. PTAL

File: SPEC.md [hunk 2]
```diff
@@ -543,6 +543,8 @@ Accessible at `$AC_METADATA_URL/acMetadata/v1/pod/hmac`
 
 ### AC Name Type
 
+Abbreviated reference as `acName`.
```
Comment (jonboulle) [inline SPEC.md]: sorry, this doesn't make sense; `acName` isn't mentioned anywhere else in the spec, and the other kinds are not references but field names. IMHO just remove this and 565 / 578

Comment (vbatts) [inline SPEC.md]: K

Comment (jonboulle) [inline SPEC.md]: ping

**Comment by jonboulle** [comment]
deprecated by #399

**Pull request closed without merging**